homeassistant/components/osoenergy/water_heater.py

```diff
@@ -34,9 +34,6 @@
         "extraenergy": STATE_HIGH_DEMAND,
     },
 }
-HEATER_MIN_TEMP = 10
-HEATER_MAX_TEMP = 80
```

Comment:
Why are these removed?

Reply:
These values are not used anymore. 
The minimum and maximum temperature of the water heater are now returned and used from the server.

Reply:
I see, but just to clarify, removing this is unrelated to this PR?

Reply:
Yes, it's just cleaning two lines of unused code.